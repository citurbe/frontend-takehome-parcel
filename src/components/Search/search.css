.search-button {
  margin-right: 10px;
  border-radius: 5px;
}

.search-box {
  margin-right: 10px;
}

input {
  width: 30%;
}
