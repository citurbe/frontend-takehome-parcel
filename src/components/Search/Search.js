import React from 'react';
import './search.css';

const Search = props => {
  return (
    <div>
      <input
        onChange={props.handleChange}
        type="text"
        value={props.searchText}
        className="search-box"
      />{' '}
      <button className="search-button" onClick={props.handleSearch}>
        Search
      </button>
      <button className="search-button" onClick={props.showSaved}>
        Show Saved
      </button>
    </div>
  );
};

export default Search;
