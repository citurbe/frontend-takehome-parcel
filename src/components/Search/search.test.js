import React from 'react';
import Search from './Search';
import renderer from 'react-test-renderer';

test('Search renders correctly', () => {
  const component = renderer.create(<Search searchText="test search text" />);

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
