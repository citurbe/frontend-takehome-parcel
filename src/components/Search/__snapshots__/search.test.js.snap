// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search renders correctly 1`] = `
<div>
  <input
    className="search-box"
    type="text"
    value="test search text"
  />
   
  <button
    className="search-button"
  >
    Search
  </button>
  <button
    className="search-button"
  >
    Show Saved
  </button>
</div>
`;
